soc/renesas/ra/ra2/Kconfig.soc

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 MUNIC SA
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA2
```

Comment:
Should be SOC_SERIES_RA2L1. This is a common issue, please fix all similar cases.

Reply:
Completed. In the new patch series.